databases/week1/homework.sql

```diff
@@ -0,0 +1,68 @@
+/*
+Find out how many tasks are in the task table
+Find out how many tasks in the task table do not have a valid due date
+Find all the tasks that are marked as done
+Find all the tasks that are not marked as done
+Get all the tasks, sorted with the most recently created first
+Get the single most recently created task
+Get the title and due date of all tasks where the title or description contains database
+Get the title and status (as text) of all tasks
+Get the name of each status, along with a count of how many tasks have that status
+Get the names of all statuses, sorted by the status with most tasks first
+ */
+select count(*)
+from task;
+
+select count(*)
+from task
+where due_date is null;
+
+select t.*
```

Comment:
It is a good practice to select only the fields in which you are interested.
```suggestion
select t.id, t.name
```